cli/app.go

```diff
@@ -61,16 +64,55 @@ Examples:
 `)
 }
 
+func promptYesNo(msg string) (result bool) {
```

Comment:
This should probably replace the implementation here as well: https://github.com/flynn/flynn/blob/40b1a276eec4c1650309cdb438460e0bc7b3eee7/cli/app.go#L83-L96